<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="0,0,0">
    <item row="0" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listWidgetFiles">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string>Choose Folder</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="RobWidget" name="imageWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_image">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>400</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>here will be an image ...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="Line" name="line">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QWidget" name="bottom" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <widget class="QWidget" name="truncated" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>149</y>
         <width>301</width>
         <height>51</height>
        </rect>
       </property>
       <widget class="QSlider" name="truncated_bar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>281</width>
          <height>29</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>none</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>10</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>truncated</string>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="object_id">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>40</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>00000001</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="type">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>152</width>
         <height>86</height>
        </rect>
       </property>
       <property name="title">
        <string>objectType</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="type_face">
          <property name="text">
           <string>Face</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="type_dontCareFace">
          <property name="text">
           <string>DontCareFace</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="occluded">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>0</y>
         <width>168</width>
         <height>142</height>
        </rect>
       </property>
       <property name="title">
        <string>occluded</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="occluded_visible">
          <property name="text">
           <string>fully visible</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="occluded_partly">
          <property name="text">
           <string>partly occluded</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="occluded_largly">
          <property name="text">
           <string>largely occluded</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="occluded_unknown">
          <property name="text">
           <string>unknown</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QListWidget" name="listObjects">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>0</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>10</y>
         <width>131</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Add new Element</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>160</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete selected</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>10</y>
         <width>321</width>
         <height>181</height>
        </rect>
       </property>
       <widget class="QTableWidget" name="tableIamges">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>321</width>
          <height>181</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="autoScrollMargin">
         <number>16</number>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>true</bool>
        </property>
        <property name="iconSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>70</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Save changes</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>100</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>try: WASD / wasd</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RobWidget</class>
   <extends>QWidget</extends>
   <header>robwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
